kibana-reports/public/components/main/report_definition_details/report_definition_details.tsx

```diff
@@ -620,7 +620,7 @@ export function ReportDefinitionDetails(props) {
           <EuiSpacer />
           {triggerSection}
           <EuiSpacer />
-          <EuiTitle>
+          {/* <EuiTitle>
```

Comment:
Can we remove this unused code?

Reply:
This will be added back once we plan to add notification. Do you want me to put a comment here?